src/rust-crypto/backup.ts

```diff
@@ -591,6 +603,173 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
     public createBackupDecryptor(decryptionKey: RustSdkCryptoJs.BackupDecryptionKey): BackupDecryptor {
         return new RustBackupDecryptor(decryptionKey);
     }
+
+    /**
+     * Restore a key backup.
+     * @param backupInfoVersion - The version of the backup info.
```

Comment:
it's the version of the backup, not the version of the info.

```suggestion
     * @param backupVersion - The backup version to restore.
```

(likewise below)